### DISCUSSION TITLE
[Idea]  Agent for ChromeOS/ Chrome tablet

### DISCUSSION THREAD

**Discussion by etherFlare** [glpi-project/glpi-agent#385] [Ideas]
### Your idea

It would be great if we can install GLPI agent in ChromeOS / Chrome tablets.

**Comment by g-bougard** [comment]
Hi @etherFlare 
I'm not an expert of ChromeOS but I may investigate that support in a near future.
First, I have one question. I see Google Store should be available on ChromeBooks. Did you look for "GLPI-Agent for Android" in it ? I really don't know if it can be run. The related project is hosted here on GH: https://github.com/glpi-project/android-inventory-agent

**Reply by g-bougard** [reply]
Then have you take a look at [ChromeBrew project](https://github.com/chromebrew/chromebrew) ? It seems you can install perl with it. If this is possible maybe you can experiment to [install GLPI-Agent from sources](https://glpi-agent.readthedocs.io/en/latest/installation/index.html#from-sources).
